api/doc/openapi.yaml

```diff
@@ -19,40 +19,6 @@ tags:
     description: Access to account resources and modules
   - name: events
     description: Access to events
-  - name: transaction_model
-    x-displayName: Transaction
-    description: |
-      <SchemaDefinition schemaRef="#/components/schemas/Transaction"/>
-  - name: account_resource_model
-    x-displayName: AccountResource
-    description: |
-      <SchemaDefinition schemaRef="#/components/schemas/MoveResource"/>
-  - name: account_module_model
-    x-displayName: AccountModule
-    description: |
-      <SchemaDefinition schemaRef="#/components/schemas/MoveModuleBytecode"/>
-  - name: event_model
-    x-displayName: Event
-    description: |
-      <SchemaDefinition schemaRef="#/components/schemas/Event"/>
-  - name: move_value_model
-    x-displayName: MoveValue
-    description: |
-      <SchemaDefinition schemaRef="#/components/schemas/MoveValue"/>
-x-tagGroups:
-  - name: API
-    tags:
-      - general
-      - transactions
-      - accounts
-      - events
-  - name: Models
-    tags:
-      - transaction_model
-      - account_resource_model
-      - account_module_model
-      - event_model
-      - move_value_model
```

Comment:
above are specific to redocly renderer, we don't need anymore after switched to https://github.com/stoplightio/elements